game.js

```diff
@@ -296,6 +335,42 @@ class Game extends Screen {
 		text(this.score, width / 2, LINE_SPACING)
 	}
 
+	drawTargetScore() {
+		textAlign(CENTER, CENTER)
+		textSize(30)
+		fill(255, 255, 250)
+		strokeWeight(4)
+		text('You need a total of ' + targetScore + ' points to win.', width / 2, LINE_SPACING * 13)
+		text('Only ' + (this.calculatePointsToGo()) + ' points to go!', width / 2, LINE_SPACING * 14)
+	}
+
+	drawProgressBar() {
+		push()
+		strokeWeight(40)
+		stroke(0, 255, 0, 100)
+		line(width / 2 - 100, LINE_SPACING + 750, width / 2 + 100, LINE_SPACING + 750)
```

Comment:
I see lots of `LINE_SPACING + 750` here.  just store it in a local var and use it so that if you need to make adjustments you only need to do in one place.  The [DRY principal](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)!